package.json

```diff
@@ -55,6 +54,7 @@
     "estree-util-visit": "^2.0.0",
     "github-slugger": "^2.0.0",
     "glob": "^11.0.3",
+    "globals": "^16.3.0",
```

Comment:
I'm unsure how up-do-date this is. It's also weird as usually _we_ define what Globals we support or not. So what if we add a new Global? What's the process?

Reply:
> I'm unsure how up-do-date this is. It's also weird as usually _we_ define what Globals we support or not. So what if we add a new Global? What's the process?

The package is updated automatically every month. Even then, the globals in V8 don't change that often.

If Node.js added a new global, it would need to be defined manually within the other object in this file. This only accounts for V8's globals.

Reply:
It's V8, well, it's the ECMAScript specification. If we build for an older version of Node.js, it won't matter. We define these types, but the older source doesn't necessarily use them.